Update Password Criteria (#5530)
Kong/docs.konghq.com: app/konnect/getting-started/access-account.md

3. Create a password.

>     The new password must be at least 8 characters long, no more than 128 characters long, and contain at least three of the

Reviewer: ```suggestion
    The new password must be between 8-128 characters, and contain at least three of the
```
